tests/functions.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class FunctionTests extends TestCase
+{
+    public function testSuccessDisplayACheese()
```

Comment:
might be worth adding a little comment above each public function with a short description of what you are hoping to achieve